examples/content/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-fonts/css/fonts.css

```diff
@@ -0,0 +1,18 @@
+/*
+ *  Copyright 2017 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+@charset "UTF-8";
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600|Asar');
```

Comment:
We'll be using "Source Sans Pro" the Adobe Open Source font. This just sets up a clientlib for including the font so that we can use it when styling the app.